deployment/ib-kubernetes-deploy.yaml

```diff
@@ -51,6 +51,10 @@ spec:
     spec:
       serviceAccountName: ib-kubernetes
       hostNetwork: true
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
```

Comment:
I think we need evaluate what is done in ovn-master yaml and see if it make sense for us [1]
[1] - https://github.com/ovn-org/ovn-kubernetes/blob/master/dist/templates/ovnkube-master.yaml.j2#L41-L65